_includes/v1.0/misc/unsupported-version.md

```diff
@@ -0,0 +1,3 @@
+{{site.data.alerts.callout_danger}}
+This version of CockroachDB is no longer supported. Please use a more recent version. 
```

Comment:
I think that including a link here to the install/download page for `/stable` would feel like we were trying to help users who're trying to use these docs.

Reply:
Done.